pylab/core/models.py

```diff
@@ -0,0 +1,52 @@
+from autoslug import AutoSlugField
+from django_extensions.db.fields import CreationDateTimeField, ModificationDateTimeField
+
+from django.contrib.auth.models import User
+from django.db import models
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext_lazy as _
+
+
+class Project(models.Model):
+    created = CreationDateTimeField()
+    modified = ModificationDateTimeField()
+    author = models.ForeignKey(User)
+    slug = AutoSlugField(populate_from='title')
+    title = models.CharField(_("Title"), max_length=255)
+    description = models.TextField(_("Description"))
+
+    def __str__(self):
+        return self.title
+
+    def get_absolute_url(self):
+        return reverse('project-details', args=[self.slug])
+
+
+class Event(models.Model):
+    OTHER = 0
+    PROJECT_DEVELOPMENT = 1
+    INITIAL_MEETING = 2
+    WEEKLY_MEETING = 3
+    EVENT_TYPE_CHOICES = (
```

Comment:
I think that we should consider creating new model for EventType. Because it looks, that EventType is quite dynamic entity, we will add Conference, Hackathon, CodingDojo and CodingWeek types, etc. This model could contain event description. E.g. weekly meeting has several parts, each of them could be described using scrum terminology, so that everybody knew how this event is structured.